File: 프로그래머스/0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.js
function solution(s) {    
    let chars = {};
    let str = []
    
    s.split("").forEach(c => {
        if(chars[c]) {
            chars[c]++;
        } else {
            chars[c] = 1;
        }
    })
    
    for(const [key, value] of Object.entries(chars)) {
        if(value === 1)
            str.push(key)
    }
    
    return str.sort().join("")
}

File: 프로그래머스/0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.py
def solution(s):
    letters = {}
    
    for c in s:
        if c not in letters.keys():
            letters[c] = 0
        letters[c] += 1
        
    str = []
    for c in letters.keys():
        if letters[c] == 1:
            str.append(c)
    
    return "".join(sorted(str))